index.html

```diff
@@ -206,59 +172,113 @@ <h3>let's connect</h3>
+              <li>
+                <img src="./Icons/Icon - Chevron - Down - Larger.svg" alt="" />
+              </li>
+            </ul>
+          </li>
+          <li class="programming-lang">
+            <ul class="web-lang">
+              <li>
+                <img src="./Icons/Ellipse 1.png" alt="js icon" /> JavaScript
+              </li>
+              <li>
+                <img src="./Icons/Ellipse 1 (1).png" alt="html icon" /> HTML
+              </li>
+              <li>
+                <img src="./Icons/Ellipse 1 (2).png" alt="css icon" /> CSS
+              </li>
+            </ul>
+          </li>
+          <li class="list-title">
+            <ul class="lists">
+              <li><h4>Frameworks</h4></li>
+              <li>
+                <img src="./Icons/Icon - Chevron - Right - Larger.svg" alt="" />
+              </li>
+            </ul>
+          </li>
+          <li>
+            <img class="list-separator" src="./Icons/Separator Bottom.svg" alt="" />
+          </li>
+          <li class="list-title">
+            <ul class="lists">
+              <li><h4>Skills</h4></li>
+              <li>
+                <img src="./Icons/Icon - Chevron - Right - Larger.svg" alt="" />
+              </li>
+            </ul>
+          </li>
+          <li>
+            <img
+              src="./Icons/Separator Bottom.svg"
+              alt=""
+              class="list-separator"
+            />
+          </li>
         </ul>
-      </section>
-    </main>
+      </div>
+    </section>
+
+    <!-- contact  -->
+    <section id="contact">
+      <div class="contact-bg-form">
+        <div class="contact-form">
+          <h1>Contact me</h1>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it!
+          </p>
+          <form action="https://formspree.io/f/xqkvalkp" method="post" id="contact-form">
+            <div class="inputs">
+              <input
+                type="text"
+                name="firstname"
+                placeholder="Abdullah Ahmadi"
+                maxlength="30"
+                required
+              />
+            </div>
+            <div class="inputs">
+              <input
+                type="email"
+                name="email"
+                id="email"
+                placeholder="abdullahahmadi019@gmail.com"
+                required
+              />
+            </div>
+            <div class="inputs-msg">
+              <textarea
+                name="msg"
+                id="msg"
+                placeholder="Write your message here..."
+                maxlength="500"
+                required
+              ></textarea>
+            </div>
+            <div id="error-message" style="display: none; color: red;"></div>
+            <button type="submit" class="contact-btn" onclick="showPopup()">Get in touch</button>
```

Comment:
- Great job with this hack :clap: :hugs: 
- [ ] I just think for separation of concerns reasons and as a matter of best practice, it is best to hold separate files for `html codes` and the `JavaScript file` to handle the logic just as you did with your `CSS styles`. Kindly implement this functionality from your `script.js` file. That way, it makes debugging easy, improves code readability, and allows for easy scalability as well as maintainability.